### ISSUE TITLE
set hint text in zefyr 

### ISSUE THREAD

**Issue by MaheshManoharan** [memspace/zefyr#486]
How to set hint text zefyr, I already put the question in Reddit and StackOverflow. but no one answers.  So tell me please

**Comment by singerdmx** [comment]
Do you mean placeholder, similar to https://github.com/singerdmx/flutter-quill/issues/11 ?

**Comment by singerdmx** [comment]
Can you share Reddit and StackOverflow links?

**Comment by singerdmx** [comment]
@MaheshManoharan I have a [change](https://github.com/singerdmx/flutter-quill/pull/34) that gets most working. Needs final touch

**Comment by MaheshManoharan** [comment]
https://www.reddit.com/r/FlutterDev/comments/lo04xy/zefyr_editor_in_flutter/
https://stackoverflow.com/questions/66289273/how-to-set-hint-text-in-zefyr-in-flutter
here is the links

**Comment by MaheshManoharan** [comment]
yes...just like placeholder... users need to know what to do in that textfield.  I don't know how to do it in zefyrEditor please help me.

**Comment by MaheshManoharan** [comment]
@singerdmx I know you did something, just like passing placeholder. But as a beginner in programming and github I don't know what to do now. Could you please say What I want to do now to get placeholder or hint text functionality?

**Comment by singerdmx** [comment]
@MaheshManoharan  OK, my [change](https://github.com/singerdmx/flutter-quill/pull/34/files) is done now. I tested it on my phone and it is working. 
All you need to do is this:
![image](https://user-images.githubusercontent.com/122956/108603779-ca1dc200-735e-11eb-9d1f-8e433fd003cb.png)

**Comment by MaheshManoharan** [comment]
@singerdmx A little bit doubt again... I am using ZefyrEditor, But you showed me in QuillEditor

**Comment by singerdmx** [comment]
I don't think Zefyr is accepting pull request.
And this change is for https://pub.dev/packages/flutter_quill
I will submit my change once my team mate verifies the change is working.

**Comment by MaheshManoharan** [comment]
I saw your package. It's nice, I will comment after check it out. Thank you for your response.

**Comment by MaheshManoharan** [comment]
I checked your example, but it shows just a white blank screen based on pub

**Comment by singerdmx** [comment]
@MaheshManoharan  the change is not checked in yet. Please wait until my team mate @li3317 verifies the change.
Then we will bump the version

**Comment by MaheshManoharan** [comment]
@singerdmx could you please inform me, when it's over.

**Comment by singerdmx** [comment]
@li3317 will update this thread here once it is in and let you know how to use placeholder with details

**Comment by li3317** [comment]
Hi @MaheshManoharan  Sorry about the delay! The change is now available on at  https://pub.dev/packages/flutter_quill! Here's an example of how it looks.
![151430107_289821215825210_5622744891969403627_n](https://user-images.githubusercontent.com/46685982/108614753-641b5400-73cb-11eb-9c86-1c9806d5271e.jpg)

**Comment by MaheshManoharan** [comment]
@li3317 , @singerdmx  Thank you both. I will comment after checking out.

**Comment by MaheshManoharan** [comment]
@li3317 , @singerdmx  No change, it just shows a blank screen. I have just used the code in pub example.

**Comment by singerdmx** [comment]
![image](https://user-images.githubusercontent.com/122956/108614917-b3558a80-73b3-11eb-83e9-3baffd32b4e5.png)


Did you add anything here?

**Comment by MaheshManoharan** [comment]
@singerdmx no I can't add placeholder, because it just shows a white blank screen. I haven't added it yet. just used this example code.

**Comment by MaheshManoharan** [comment]
@singerdmx 
how did you upload the image? I can post my screenshot.

**Comment by li3317** [comment]
New version with placeholder support is published!

**Comment by MaheshManoharan** [comment]
@li3317 it shows awaiting... 
![Screenshot_1613878751](https://user-images.githubusercontent.com/69787389/108615080-5b0ba080-7426-11eb-8c71-366965ee93c6.png)

**Comment by MaheshManoharan** [comment]
I got above screen - blank white screen

**Comment by singerdmx** [comment]
Paste your code

**Comment by singerdmx** [comment]
Also can you try to run the code in https://github.com/singerdmx/flutter-quill/tree/stable/app and let us know what you see?

**Comment by MaheshManoharan** [comment]
`import 'dart:async';
import 'dart:io';

import 'package:flutter/material.dart';
import 'package:flutter_quill/widgets/controller.dart';
import 'package:flutter_quill/widgets/editor.dart';
import 'package:flutter_quill/widgets/toolbar.dart';

void main() => MaterialApp(home: HomePage());

class HomePage extends StatefulWidget {
  @override
  _HomePageState createState() => _HomePageState();
}

class _HomePageState extends State<HomePage> {
  QuillController _controller = QuillController.basic();

  @override
  Widget build(BuildContext context) {
    return Scaffold(
        body: Column(
      children: [
        QuillToolbar.basic(
            controller: _controller, uploadFileCallback: _uploadImageCallBack),
        Expanded(
          child: Container(
            child: QuillEditor.basic(
              controller: _controller,
              readOnly: false, // change to true to be view only mode
            ),
          ),
        )
      ],
    ));
  }

  Future<String> _uploadImageCallBack(File file) async {
    // call upload file API and return file's absolute url
    return new Completer<String>().future;
  }
}
`

**Comment by singerdmx** [comment]
You did it wrong
please read REAME

**Comment by singerdmx** [comment]
Please follow https://github.com/singerdmx/flutter-quill/blob/stable/app/lib/pages/home_page.dart

**Comment by MaheshManoharan** [comment]
oh..my bad. I have been using zefyr, now I have changed my mind to use flutter-quill, Your response is the best. Thank you, guys. You are so supportive. flutter-quill/stable app code is working fine.  Is there any doubt or question, should I comment here. As a beginner, I don't know what to talk.

**Comment by singerdmx** [comment]
@MaheshManoharan for issues regarding https://pub.dev/packages/flutter_quill , please create issue under https://github.com/singerdmx/flutter-quill/issues

We were using Zefyr until we found it uses custom delta format that is incompatible with quilljs https://quilljs.com/docs/formats

Also lack of support is one of the reasons as well.

**Comment by MaheshManoharan** [comment]
okay.. is it free to use in commercial projects?

**Comment by singerdmx** [comment]
Sure, I don't care how you use it as long as it helps you

**Comment by MaheshManoharan** [comment]
@singerdmx cool...

**Comment by cgestes** [comment]
@singerdmx wrong bug tracker. Please keep your project related discussion in your project.

Still its ok to mention your project here, especially as inspiration to how to do thinks. :)
